src/Apps/Artwork/Components/ArtworkSidebar/ArtworkSidebarCurrentBidInfo.tsx

```diff
@@ -20,9 +21,18 @@ export interface ArtworkSidebarCurrentBidInfoProps {
   artwork: ArtworkSidebarCurrentBidInfo_artwork
 }
 
+@track()
 export class ArtworkSidebarCurrentBidInfo extends React.Component<
   ArtworkSidebarCurrentBidInfoProps
 > {
+  @track(() => {
+    return {
+      context_module: Schema.ContextModule.Sidebar,
+      action_type: Schema.ActionType.Click,
+      subject: Schema.Subject.AuctionBuyerPremium,
+      type: Schema.Type.Link,
+    }
+  })
   onClickBuyerPremium(mediator) {
```

Comment:
`handleBuyerPremiumClick` (on = incoming props passed to component; handle = internal event handler)